scripts/utilities/update_revision.sh

```diff
@@ -94,13 +154,57 @@ send_to_review ()
 {
 	print_msg 1 "Preparing commit to send for review."
 
-	# Check connection to GitHub
+	# Check connection to GitHub or Gerrit
 	#
-	print_msg 2 "Checking connection to GitHub."
-	ssh git@github.com > /dev/null 2>&1
-	if [[ $? -ne 1 ]]; then
-		print_msg 0 "Cannot connect to GitHub."
-		return 1
+	if [[ ${service} = "github" ]];
```

Comment:
It would be better to have a function for each service, this generic send_to_review function could have the common code and call the specific function.